internal/api/ui/login/static/resources/scripts/init_password_check.js

```diff
@@ -1,14 +1,15 @@
 function CheckInitPwPolicy() {
-    let policyElement = document.getElementById("password");
-    let pwNew = policyElement.value;
-    let pwNewConfirmation = document.getElementById("passwordconfirm").value;
+  let pwNew = document.getElementById("password");
+  let pwNewValue = pwNew.value;
+  const pwNewConfirmation = document.getElementById("passwordconfirm");
+  let pwNewConfirmationValue = pwNewConfirmation.value;
 
-    if (ComplexityPolicyCheck(policyElement, pwNew, pwNewConfirmation) === false) {
-        return false;
-    }
+  if (ComplexityPolicyCheck(pwNew) === false) {
```

Comment:
same as comment above

Reply:
and the checks look like the same as in change_password_check.js

Reply:
I'd not use a script for multiple pages as this could be not comprehensible for additional changes..